lib/chef/azure/commands/enable.rb

```diff
@@ -173,6 +173,7 @@ def configure_chef_only_once
         config[:node_ssl_verify_mode] =  bootstrap_options['node_ssl_verify_mode'] if bootstrap_options['node_ssl_verify_mode']
         runlist = @run_list.empty? ? [] : escape_runlist(@run_list)
         load_cloud_attributes_in_hints if ! @ohai_hints.empty?
+        config[:first_boot_attributes] = eval(@first_boot_attributes)

```

Comment:
@btm here eval is used to convert json string into Hash the the @first_boot_attributes has contains string e.g. "{\"name\"=>\"testjson\"}" to convert this in {"name"=>"testson"} eval is used which is actually a dirty way. I am checking if there is anything better to do this.